### PR TITLE
[PPML] Change folder directory for ref/base image

### PR DESCRIPTION
**Pull request by gc-fu**
## Description
As instructed, move all the example files and related patches to the base image.

Remove SGX_MEM_SIZE argument in bigdl-ppml-gramine-base image.

### CODE REVIEW

File: ppml/trusted-deep-learning/ref/Dockerfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ ARG BASE_IMAGE_TAG
 # stage 1. generate SGX secrets and prepare KMS env
 FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG as temp
 
-ARG SGX_MEM_SIZE
-ARG SGX_LOG_LEVEL
+ARG SGX_MEM_SIZE     32G
+ARG SGX_LOG_LEVEL    error
```
Comment (ShanSimu) [inline ppml/trusted-deep-learning/ref/Dockerfile:8]: default value?

**Review by ShanSimu — COMMENTED** [review]
lgtm

File: ppml/trusted-deep-learning/ref/Dockerfile (hunk 1 above)
Comment (gc-fu) [reply ppml/trusted-deep-learning/ref/Dockerfile:8]: yep

**Review by Hanyu-Jin — APPROVED** [review]
LGTM

**Pull request merged**